### PR TITLE
adding more properties to pass down to vmwatch

### PR DESCRIPTION
**Pull request by manishsat**
Adding below properties:
1. DisableConfigReader: To disable config reader in vmwatch
2. GlobalConfigUrl: To over ride config url from which vmwatch reads the global properties from, this is mostly use in E2E
3. execution-environment: sending app health execution environment (Test|Prod)

### CODE REVIEW

File: main/handlersettings.go
Comment (klugorosado) [inline main/handlersettings.go:1]: Can customers set this fields? If so I believe the vmWatchSettings schema in schema.go should be modified.

File: main/handlersettings.go [hunk 1]
```diff
@@ -86,17 +86,19 @@ func (h handlerSettings) validate() error {
...
-	Enabled                bool                   `json:"enabled,boolean"`
-	SignalFilters          *vmWatchSignalFilters  `json:"signalFilters"`
-	ParameterOverrides     map[string]interface{} `json:"parameterOverrides,object"`
-    EnvironmentAttributes  map[string]interface{} `json:"environmentAttributes,object"`
+	Enabled               bool                   `json:"enabled,boolean"`
+	SignalFilters         *vmWatchSignalFilters  `json:"signalFilters"`
+	ParameterOverrides    map[string]interface{} `json:"parameterOverrides,object"`
+	EnvironmentAttributes map[string]interface{} `json:"environmentAttributes,object"`
+	GlobalConfigUrl       string                 `json:"globalConfigUrl,string"`
+	DisableConfigReader   bool                   `json:"disableConfigReader,boolean"`
```
Comment (klugorosado) [inline main/handlersettings.go:101]: Can customers set this fields? If so I believe the vmWatchSettings schema in schema.go should be modified.

File: main/vmWatch.go [hunk 2]
```diff
@@ -36,7 +37,10 @@ const (
 
 const (
 	AllowVMWatchCgroupAssignmentFailureVariableName string = "ALLOW_VMWATCH_CGROUP_ASSIGNMENT_FAILURE"
-	RunningInDevContainerVariableName string = "RUNNING_IN_DEV_CONTAINER"
+	RunningInDevContainerVariableName               string = "RUNNING_IN_DEV_CONTAINER"
+	AppHealthExecutionEnvironmentProd                      = "Prod"
+	AppHealthExecutionEnvironmentTest                      = "Test"
+	AppHealthPublisherNameTest                             = "Microsoft.ManagedServices.Edp"
```
Comment (klugorosado) [inline main/vmWatch.go:43]: nit: add type string to the rest
```suggestion
	AllowVMWatchCgroupAssignmentFailureVariableName string = "ALLOW_VMWATCH_CGROUP_ASSIGNMENT_FAILURE"
	RunningInDevContainerVariableName               string = "RUNNING_IN_DEV_CONTAINER"
	AppHealthExecutionEnvironmentProd                      = "Prod"
	AppHealthExecutionEnvironmentTest                      = "Test"
	AppHealthPublisherNameTest                             = "Microsoft.ManagedServices.Edp"
```

**Review by dpoole73 — APPROVED** [review]

**Review by klugorosado — APPROVED** [review]
looks good

**Pull request merged**